docs/docs/guide/platform/README.md

```diff
@@ -44,50 +44,38 @@ If you're using `react`, DXOS provides a simple [UI flow](../react/spaces#joinin
 
 ## Glossary
 
 See the [glossary](../glossary) for definitions of other terms you'll find in this guide.
 
 ## How to use ECHO
 
-*   Install the appropriate npm package [`@dxos/client`](../typescript) or [`@dxos/react-client`](../react)
-*   Create a [Client](../typescript#configuration) (or a [ClientProvider](../react#cofiguration) in react)
-*   Set up an identity with [HALO](./halo)
-*   Create or Join a [Space](#spaces)
-*   [Query objects](../typescript/queries) (in [react](../react/queries))
-*   [Create objects](../typescript/mutations#creating-objects) (in [react](../react/mutations))
-*   [Mutate objects](../typescript/mutations) (in [react](../react/mutations))
+- Install the appropriate npm package [`@dxos/client`](../typescript) or [`@dxos/react-client`](../react)
+- Create a [Client](../typescript#configuration) (or a [ClientProvider](../react#cofiguration) in react)
+- Set up an identity with [HALO](./halo)
+- Create or Join a [Space](#spaces)
+- [Query objects](../typescript/queries) (in [react](../react/queries))
+- [Create objects](../typescript/mutations#creating-objects) (in [react](../react/mutations))
+- [Mutate objects](../typescript/mutations) (in [react](../react/mutations))
 
-## Local Vault Topology
+## Vault
 
-ECHO is uniquely organized to give control over information to the end-user and their devices.
+ECHO is designed to allow users to own their own data. The ECHO Vault is a secure storage mechanism, responsible for holding end-user identity (keys, credentials, metadata, ..., etc.) and all ECHO data in persistent browser or disk storage. The vault serves as an identity wallet and data vault where specific devices or applications can be revoked from accessing user data at any time.
 
-Browsers isolate web apps running on different domains, and this property is used to isolate the main `vault` of storage to a specific domain. By default that's [HALO](./halo) on `halo.dxos.org` but developers are free to [operate their own](../typescript/config#custom-halo-source) copies of HALO on their local machine (e.g. using [KUBE](./kube)), their local network, or any other domain.
+Browsers provide storage APIs that allow applications to store data locally. Storage for web apps is isolated to the web app's domain, and this property is used to isolate the main vault of storage to the DXOS application's domain.
 
-The vault domain is responsible for holding end-user identity (keys, credentials, metadata, ..., etc.) and all ECHO data in persistent browser or disk storage. It exposes this data to applications via an API over [`postMessage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage).
+In order to allow multiple applications to access the same vault, the user must initiate a device invitation, which synchronizes the vault across both applications, giving them both read and write access to all of the user's data stored in the vault. See [Device Invitations](./halo#device-invitations) for more information.
```

Comment:
"access the same vault" based on the above definition of "vault" no two apps can ever access the same vault because that would be cross-origin access.

Reply:
the vault was intended more of a conceptual thing that spans any origin/app that the user invites